FrameworkSystem/scripts/dirac-stop-component.py

```diff
@@ -14,6 +12,11 @@
                                     '  service|agent: Name of the particular component (default *: all)' ] ) )
 Script.parseCommandLine()
 args = Script.getPositionalArgs()
+
+from DIRAC.FrameworkSystem.Client.ComponentInstaller import gComponentInstaller
```

Comment:
I am fixing the others, but in this file it is already after the `parseCommandLine()` :)